ui/apps/platform/src/routePaths.ts

```diff
@@ -102,6 +106,119 @@ export const vulnerabilitiesWorkloadCveDeploymentSinglePath = `${vulnerabilities
 export const vulnerabilityReportsPath = `${vulnerabilitiesBasePath}/reports`;
 export const vulnerabilityReportPath = `${vulnerabilitiesBasePath}/reports/:reportId`;
 
+// Source of truth for conditional rendering of Body route paths and NavigationSidebar links.
+
+type RouteDescription = {
+    featureFlagDependency?: FeatureFlagEnvVar; // can add array as alternative if needed in the future
```

Comment:
We have had multiple feature flag dependencies for a single route in the recent past (Collection IIRC). Any harm in making this an array now?

Reply:
Good point. I will do it in the next step.